tests/client/api_restful/test_outputs_id.c

```diff
@@ -15,31 +15,27 @@ void tearDown(void) {}
 
 void test_deser_outputs() {
   // empty output ids
-  char const* const data_empty =
-      "{\"limit\":1000,"
-      "\"count\":0,\"data\":[],\"ledgerIndex\":837834}";
+  char const* const data_empty = "{\"pageSize\":2,\"items\":[],\"ledgerIndex\":837834}";
 
   res_outputs_id_t* res = res_outputs_new();
   TEST_ASSERT_NOT_NULL(res);
   TEST_ASSERT(deser_outputs(data_empty, res) == 0);
   TEST_ASSERT(res->is_error == false);
-  TEST_ASSERT(res->u.output_ids->limit == 1000);
-  TEST_ASSERT(res->u.output_ids->count == 0);
+  TEST_ASSERT(res->u.output_ids->page_size == 2);
   TEST_ASSERT(utarray_len(res->u.output_ids->outputs) == 0);
   TEST_ASSERT(res->u.output_ids->ledger_idx == 837834);
   res_outputs_free(res);
   res = NULL;
 
-  // with output ids
+  // with output ids and without cursor
   char const* const data_1 =
-      "{\"limit\":1000,"
-      "\"count\":2,\"data\":[\"1c6943b0487c92fd057d4d22ad844cc37ee27fe6fbe88e5ff0d20b2233f75b9d0005\","
+      "{\"pageSize\":2,\"items\":[\"1c6943b0487c92fd057d4d22ad844cc37ee27fe6fbe88e5ff0d20b2233f75b9d0005\","
       "\"ed3c3f1a319ff4e909cf2771d79fece0ac9bd9fd2ee49ea6c0885c9cb3b1248c0010\"],\"ledgerIndex\":837834}";
   res = res_outputs_new();
   TEST_ASSERT_NOT_NULL(res);
   TEST_ASSERT(deser_outputs(data_1, res) == 0);
   TEST_ASSERT(res->is_error == false);
-  TEST_ASSERT(res->u.output_ids->limit == 1000);
+  TEST_ASSERT(res->u.output_ids->page_size == 2);
   TEST_ASSERT(res_outputs_output_id_count(res) == 2);
   TEST_ASSERT(utarray_len(res->u.output_ids->outputs) == 2);
   TEST_ASSERT_EQUAL_MEMORY("1c6943b0487c92fd057d4d22ad844cc37ee27fe6fbe88e5ff0d20b2233f75b9d0005",
```

Comment:
This validates data without checking for NULL string terminator. Because message id is a string, it is more appropriate to use `TEST_ASSERT_EQUAL_STRING`.

Reply:
Done.